test/hotspot/gtest/runtime/test_os_windows.cpp

```diff
@@ -135,8 +135,20 @@ static void create_rel_directory_w(const wchar_t* path) {
 static void delete_empty_rel_directory_w(const wchar_t* path) {
   WITH_ABS_PATH(path);
   EXPECT_TRUE(file_exists_w(abs_path)) << "Can't delete directory: \"" << path << "\" does not exists";
-  BOOL result = RemoveDirectoryW(abs_path);
-  EXPECT_TRUE(result) << "Failed to delete directory \"" << path << "\": " << GetLastError();
+  const int max_wait_time = 20;
```

Comment:
This isn't a "wait time" it is a retry count.

Reply:
Right, I've changed it to retry_count.